coremis_app_integration/endpoints/login.py

```diff
@@ -0,0 +1,59 @@
+from django.utils.translation import gettext_lazy as _
+from rest_framework.authentication import authenticate
+from rest_framework.decorators import (
+    authentication_classes,
+    permission_classes,
+    api_view
+)
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+from rest_framework import status
+
+from coremis_app_integration.authorization import (
+    CustomBasicAuthentication,
+    DETAILS,
+    FAILED,
+    SUCCESS
+)
+from payroll.models import PaymentPoint
+
+
+@api_view(['POST'])
+@authentication_classes([CustomBasicAuthentication])
+@permission_classes([IsAuthenticated])
+def login(request):
+    # check if user is assigned to any PaymentPoint entity which indicates ppm user role
+    username = request.data.get('username', None)
+    password = request.data.get('password', None)
+    authenticated_user = authenticate(username=username, password=password)
+    if authenticated_user:
```

Comment:
Instead of adding nested if statements it's better to do something like: 

```
if not authenticated_user:
    return error 
...
```

it has higher readability

Reply:
done